docs/source/membership/membership.md

```diff
@@ -50,62 +94,200 @@ multiple MSPs and name them accordingly, e.g., `ORG2-MSP-NATIONAL` and
+
+![MSP1c](./ca-msp-visualization.png)
+
+The resulting ROLE and OU attributes are visible inside the X.509
+signing certificate located in the /signcerts folder. The `ROLE`
+attribute is identified as `hf.Type` and  refers to an actor's role
+within its organization, (specifying, for example, that an actor is a
+`peer`). See the following snippet from a signing certificate shows how the
+Roles and OUs are represented in the certificate.
+
+![MSP1d](./signcert.png)
 
 Finally, though this is a slight misuse of OUs, they can sometimes be used by
 different organizations in a consortium to distinguish each other. In such cases, the
 different organizations use the same Root CAs and Intermediate CAs for their chain
 of trust, but assign the `OU` field to identify members of each organization.
 We'll also see how to configure MSPs to achieve this later.
 
-### Local and Channel MSPs
+## MSP domains
 
-MSPs appear in two types of places in a blockchain network:
+MSPs occur in two domains in a blockchain network:
 
-* In channel configuration (**channel MSP**)
 * Locally on an actor's node (**local MSP**)
+* In channel configuration (**channel MSP**)
+
+### Local MSPs
 
-**Local MSPs are defined for clients (users) and for nodes (peers and orderers)**.
-Local MSPs define the permissions for that node (who the peer admins are, for example).
-The local MSPs of the users allow the user side to authenticate itself in its transactions
-as a member of a channel (e.g. in chaincode transactions), or as the owner of a
-specific role into the system (an org admin, for example, in configuration transactions).
+**Local MSPs are defined for clients and for nodes (peers and
+orderers)**.
+Local MSPs define the permissions for that node (who the peer admins
+are, for example). The local MSPs of clients (the account holders in
+the banking scenario above), allow the user to authenticate
+itself in its transactions as a member of a channel (e.g. in chaincode
+transactions), or as the owner of a specific role into the system (an
+org admin, for example, in configuration transactions).
 
-**Every node and user must have a local MSP defined**, as it defines who has
-administrative or participatory rights at that level (peer admins will not necessarily
-be channel admins, and vice versa).
+**Every node must have a local MSP defined**, as it defines
+who has administrative or participatory rights at that level (peer
+admins will not necessarily be channel admins, and vice versa).  This
+allows for authenticating member messages outside the context of a
+channel and to define the permissions over a particular node
+(who has the ability to install chaincode on a peer, for example).
+
+An orderer local MSP is also defined on the file system of the node
+and only applies to that node. Like peer nodes, orderers are also
+owned by a single organization and therefore have a single MSP to
+list the actors or nodes it trusts.
```

Comment:
Might also want to add a note here about ordering services that are owned by multiple organizations, as it will be a useful way to delineate some of these concepts. 

* Each node has a local MSP, defining the admins of that node. One or more nodes will be owned by an organization. This organization has an MSP defining its admins. The organization, the admin of the organization, the admin of the node, and the node itself should all have the same root of trust. 

* These ordering service organizations collectively run the system channel, which has a channel MSP, in which the organizations collectively make decisions relevant to the system channel (managing the consortium, mostly importantly, and the default policies which by default are inherited during the creation of a channel).

Reply:
incorporated this text into the section.